docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1564 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        assumed_role_object = sts.assume_role(
</pre><p>Call it the "DAAC access role"? not New role</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>